tests/services/detectionlists/test_user_profile.py

```diff
@@ -242,3 +266,17 @@ def test_refresh_posts_expected_data(
             posted_data["tenantId"] == user_context.get_current_tenant_id()
             and posted_data["userId"] == "942897397520289999"
         )
+
+    def test_add_cloud_alias_limit_raises_custom_error_on_limit(
+        self,
+        mock_connection,
+        user_context,
+        mock_user_client_error_on_adding_cloud_aliases,
+    ):
+        detection_list_user_client = DetectionListUserService(
+            mock_connection,
+            user_context,
+            mock_user_client_error_on_adding_cloud_aliases,
+        )
+        with pytest.raises(Py42CloudAliasLimitExceeded):
```

Comment:
Additionally, you can verify the error message by doing this:

```
with pytest.raises(Py42CloudAliasLimitExceeded) as err:
   detection_list_user_client.add_cloud_alias("942897397520289999", "Test")

assert ""Cloud alias limit exceeded." in str(err.value)
```

Reply:
added.